protocol/rpc_status.go

```diff
@@ -18,31 +18,24 @@
 package protocol
 
 import (
+	"github.com/apache/dubbo-go/common/constant"
+	"github.com/apache/dubbo-go/common/logger"
```

Comment:
OMG， move it to the 3rd import block, pls.

Reply:
done